open_academy/models/session.py

```diff
@@ -1,14 +1,47 @@
-from odoo import models, fields
+from datetime import date
+
+from odoo import models, fields, api, _
```

Comment:
```suggestion
from odoo import _, api, fields, models
```

Include isort on your tools to keep the dependencies properly organized